packages/is-shallow-equal/package.json

```diff
@@ -26,14 +26,6 @@
 	"dependencies": {
 		"@babel/runtime": "^7.0.0"
 	},
-	"devDependencies": {
-		"benchmark": "^2.1.4",
-		"fbjs": "^0.8.16",
-		"is-equal-shallow": "^0.1.3",
-		"shallow-equal": "^1.0.0",
```

Comment:
Maybe these should have been made as `optionalDependencies` 😅 They're only needed for the benchmark. Caused some confusion for me looking at `package.json` changes.

Reply:
From https://docs.npmjs.com/files/package.json#optionaldependencies:
> Entries in `optionalDependencies` will override entries of the same name in `dependencies`, so it’s usually best to only put in one place.

That might work, too. Benchmarking code is unfortunately very tricky in this context. You don't want to make packages published to npm to have those dependencies to be installed. On the other hand, it's confusing to keep them in the root `package.json` file. Maybe we should move benchmarks to their own private package?

Reply:
I think it's such a rare use-case too that I'd be fine with them not being referenced as dependencies anywhere, and then the person running them would just install them each manually.